src/commands/template.rs

```diff
@@ -150,27 +151,20 @@ fn print_containerfile(containerfile: &str) -> String {
         process::exit(1);
     });
 
-    trace!("Containerfile contents {path}:\n{file}");
+    debug!("Containerfile contents {path}:\n{file}");
 
     file
 }
 
-fn get_module_from_file(file_name: &str) -> String {
-    trace!("get_module_from_file({file_name})");
-
-    let io_err_fn = |e| {
-        error!("Failed to read module {file_name}: {e}");
-        process::exit(1);
-    };
+#[must_use]
+pub fn template_module_from_file(file_name: &str) -> String {
+    debug!("get_module_from_file({file_name})");
 
     let file_path = PathBuf::from("config").join(file_name);
-
-    let file = fs::read_to_string(file_path).unwrap_or_else(io_err_fn);
-
-    let serde_err_fn = |e| {
-        error!("Failed to deserialize module {file_name}: {e}");
-        process::exit(1);
-    };
+    let file = fs::read_to_string(file_path).unwrap_or_else(|e| {
+        error!("Failed to read module {file_name}: {e}");
+        String::default()
+    });
```

Comment:
This is cleaner, but this allows templating to continue. I guess the question is do we want a user to be notified immediately by a missing file, or just let it continue? I personally would like the program to exit if the file doesn't exist.

Reply:
yeah it was mostly for testing at the time